### PR TITLE
i18n(es): update data fetching

### PR DESCRIPTION
**Pull request by kevinzunigacuellar**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- Rename page in `nav.ts`
- Update `data-fetching.md` #1245

### CODE REVIEW

File: src/pages/es/guides/data-fetching.md [hunk 1]
```diff
@@ -26,19 +26,49 @@ const data = await response.json();
...
+<h1>Usuario</h1>
 <h2>{randomUser.name.first} {randomUser.name.last}</h2>
 
 <!-- Los datos obtenidos en la compilación pueden ser pasados como props a otros componentes -->
 <Contact client:load email={randomUser.email} />
 <Location city={randomUser.location.city} />
 ```
 
+:::note
+Recuerde, todos los datos en los componentes de Astro se obtienen cuando se renderiza el componente.
```
Comment (agustinmulet) [inline src/pages/es/guides/data-fetching.md:38]:
```suggestion
Recuerda, todos los datos en los componentes de Astro se obtienen cuando se renderiza el componente.
```

File: src/pages/es/guides/data-fetching.md [hunk 2]
```diff
@@ -26,19 +26,49 @@ const data = await response.json();
...
+
+Cuando tu proyecto de Astro es desplegado obtendrá los datos **una vez, en el momento de la compilación**. En desarrollo, verás el fetching de datos al actualizar los componentes. Si necesitas hacer fetching datos varias veces del lado del cliente, use un [componente de framework](/es/core-concepts/framework-components/) o un [script del lado del cliente](/es/core-concepts/astro-components/#scripts-del-lado-del-cliente) en el componente de Astro.
```
Comment (agustinmulet) [inline src/pages/es/guides/data-fetching.md:40]:
```suggestion
Cuando tu proyecto de Astro es desplegado obtendrá los datos **una vez, en el momento de la compilación**. En desarrollo, verás el fetching de datos al actualizar los componentes. Si necesitas hacer fetching datos varias veces del lado del cliente, usa un [componente de framework](/es/core-concepts/framework-components/) o un [script del lado del cliente](/es/core-concepts/astro-components/#scripts-del-lado-del-cliente) en el componente de Astro.
```

File: src/pages/es/guides/data-fetching.md [hunk 3]
```diff
@@ -26,19 +26,49 @@ const data = await response.json();
...
+:::
+
+## `fetch()` en componentes de framework
+
+La función `fetch()` también está disponible globalmente para cualquier [componentes de framework](/es/core-concepts/framework-components/):
```
Comment (agustinmulet) [inline src/pages/es/guides/data-fetching.md:45]:
```suggestion
La función `fetch()` también está disponible globalmente para cualquier [componente de framework](/es/core-concepts/framework-components/):
```

File: src/pages/es/guides/data-fetching.md [hunk 4]
```diff
@@ -67,38 +97,76 @@ const response = await fetch("https://graphql-weather-api.herokuapp.com",
...
 <p>Clima: {weather.getCityByName.weather.summary.description}</p>
 ```
 
-:::note
-Recuerde, todos los datos en los componentes de Astro se recuperan cuando el componente es renderizado.
+## Obtener desde un Headless CMS
 
-Al desplegar su proyecto, Astro solicitará los datos **una sola vez en el momento de la compilación**. En desarrollo, verá nuevas solicitudes de datos al actualizar los componentes. Si necesita volver a obtener datos varias veces del lado del cliente, use un [componente de framework](/es/core-concepts/framework-components/) o un [script del lado del cliente](/es/core-concepts/astro-components/#scripts-del-lado-del-cliente) en un componente de Astro.
-:::
+¡Obtenga contenido remoto de tu CMS favorito como Storyblok o WordPress!
```
Comment (agustinmulet) [inline src/pages/es/guides/data-fetching.md:107]:
```suggestion
¡Obtén contenido remoto de tus CMS favoritos como Storyblok o WordPress!
```

File: src/pages/es/guides/data-fetching.md [hunk 5]
```diff
@@ -67,38 +97,76 @@ const response = await fetch("https://graphql-weather-api.herokuapp.com",
...
 
-## `fetch()` en componentes de framework
+Los componentes de Astro pueden obtener datos de tu CMS y luego renderizarlos en la página. Usando [rutas dinámicas](/es/core-concepts/routing/#rutas-dinámicas), los componentes pueden incluso generar páginas basadas en el contenido CMS.
```
Comment (agustinmulet) [inline src/pages/es/guides/data-fetching.md:109]: Pequeña sugerencia, suena mejor así? 🤔 

```suggestion
Los componentes de Astro pueden obtener datos de tu CMS y luego renderizarlos en la página. Usando [rutas dinámicas](/es/core-concepts/routing/#rutas-dinámicas), los componentes pueden incluso generar páginas basadas en el contenido obtenido del CMS.
```

File: src/pages/es/guides/data-fetching.md [hunk 6]
```diff
@@ -67,38 +97,76 @@ const response = await fetch("https://graphql-weather-api.herokuapp.com",
...
+      <li><a href={link.slug}>{link.name}</a></li>
+    ))}
+  </ul>
+</BaseLayout>
+```
 
-// Los componentes que se renderizan en tiempo de compilación también se registran en la CLI.
-// Cuando se representan con una directiva client:*, también se registran en la consola del navegador.
-console.log(data);
+Vea el tutorial completo [Agregar un Headless CMS a Astro en 5 minutos](https://www.storyblok.com/tp/add-a-headless-cms-to-astro-in-5-minutes) para agregar Storyblok a tu proyecto de Astro!
```
Comment (agustinmulet) [inline src/pages/es/guides/data-fetching.md:140]:
```suggestion
¡Vea el tutorial completo [Agregar un Headless CMS a Astro en 5 minutos](https://www.storyblok.com/tp/add-a-headless-cms-to-astro-in-5-minutes) para agregar Storyblok a tu proyecto de Astro!
```

File: src/pages/es/guides/data-fetching.md [hunk 7]
```diff
@@ -67,38 +97,76 @@ const response = await fetch("https://graphql-weather-api.herokuapp.com",
...
-import { h } from 'preact';
+### Ejemplo: Storyblok API
 
-const data = await fetch('https://example.com/movies.json').then((response) =>
-  response.json()
-);
+```astro
+---
+// src/pages/index.astro
+// Obtenga la lista de enlaces de tu página de Storyblok usando @storyblok/js
```
Comment (agustinmulet) [inline src/pages/es/guides/data-fetching.md:118]:
```suggestion
// Obtén la lista de enlaces de tu página de Storyblok usando @storyblok/js
```

File: src/pages/es/guides/data-fetching.md [hunk 8]
```diff
@@ -67,38 +97,76 @@ const response = await fetch("https://graphql-weather-api.herokuapp.com",
 const json = await response.json();
 const weather = json.data
 ---
-<h1>Obtenega datos sobre el clima en el momento de la compilación</h1>
+<h1>Obtenga datos sobre el clima en el momento de la compilación</h1>
```
Comment (agustinmulet) [inline src/pages/es/guides/data-fetching.md:100]:
```suggestion
<h1>Obtén datos sobre el clima en el momento de la compilación</h1>
```

**Review by agustinmulet — COMMENTED** [review]
Minor changes, keep up the good work! 🧑‍🚀

**Comment by Yan-Thomas** [comment]
Thanks @kevinzunigacuellar, @agustinmulet 💜

**Pull request merged**